src/main/resources/application.yaml

```diff
@@ -68,6 +68,9 @@ quarkus:
   banner:
     path: banner/banner.txt
 
+  log:
```

Comment:
Is this intentional? I think we try to keep settings in alphabetic order.
(I can see it was moved from being under `http` so that was necessary)

Reply:
I see. I moved it where it made logical sense to me, was not aware of alpha order convention